CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 * Unified the `run_conversion` method of `BaseSegmentationExtractorInterface` with that of all the other base interfaces. The method `write_segmentation` now uses the common `make_or_load_nwbfile` context manager [PR #29](https://github.com/catalystneuro/neuroconv/pull/29)
 * Coerced the recording extractors with `spikeextractors_backend=True` to BaseRecording objects for Axona, Blackrock, Openephys, and SpikeGadgets. [PR #38](https://github.com/catalystneuro/neuroconv/pull/38)
 
+## Documentation and tutorial enhancements:
```

Comment:
@CodyCBakerPhD 
created a new category in the changelog for documentation (followed pynwb's style) let me know if I should rollback, otherwise it should be done.

Reply:
Cool, thanks!